Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
On Fri, Jun 03, 2016 at 11:36:46AM -0700, Stephen Day wrote:
> > -	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
> > +	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
> 
> `-e` isn't supported on BSD userspace tools. Removing it didn't change the output.

Maybe not for your echo implementation.  But the \n bits are not in
POSIX, so whether or not they're supported is probably undefined.  On
my system:

  $ command -V echo
  echo is a shell builtin
  $ echo $SHELL
  /bin/bash
  $ echo $BASH_VERSION
  4.3.42(1)-release
  $ echo 'a\n\nb'
  a\n\nb